atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitionsTest.java

```diff
@@ -61,4 +61,21 @@ public void cfDefWithDifferingGcGraceSecondsValuesShouldNotMatch() {
         assertFalse("ColumnDefinitions with different gc_grace_seconds should not match",
                 ColumnFamilyDefinitions.isMatchingCf(clientSideTable, clusterSideTable));
     }
+
+    @Test
+    public void identicalCfsAreEqual() {
+        CfDef cf1 = ColumnFamilyDefinitions.getCfDef(
+                "test_keyspace",
+                TableReference.fromString("test_table"),
+                FOUR_DAYS_IN_SECONDS,
+                AtlasDbConstants.GENERIC_TABLE_METADATA);
+
+        CfDef cf2 = ColumnFamilyDefinitions.getCfDef(
+                "test_keyspace",
+                TableReference.fromString("test_table"),
+                FOUR_DAYS_IN_SECONDS,
+                AtlasDbConstants.GENERIC_TABLE_METADATA);
+
```

Comment:
nit: Maybe add a test where `isAppendHeavyAndReadLight` is set for the cfdef.

Reply:
that's fair, I'll add one